Source/ACE.Server/Managers/LandblockManager.cs

```diff
@@ -59,24 +60,28 @@ public static List<Landblock> GetActiveLandblocks()
         }
 
         /// <summary>
-        /// This should only be used for debugging/development purposes.
+        /// Loads the specified list of landblocks and optionally their adjacents.
         /// </summary>
-        public static void ForceLoadLandBlock(LandblockId blockid)
+        public static void ForceLoadLandBlock(LandblockId blockid, bool propagate = false)
         {
-            GetLandblock(blockid, false);
+            GetLandblock(blockid, propagate, true);
         }
 
         /// <summary>
         /// gets the landblock specified, creating it if it is not already loaded.  will create all
         /// adjacent landblocks if propagate is true (outdoor world roaming).
         /// </summary>
-        private static Landblock GetLandblock(LandblockId landblockId, bool propagate)
+        private static Landblock GetLandblock(LandblockId landblockId, bool propagate, bool permaload = false)
         {
             lock (landblockMutex)
             {
                 var landblock = landblocks[landblockId.LandblockX, landblockId.LandblockY];
                 var autoLoad = propagate && landblockId.MapScope == MapScope.Outdoors;
 
+                // Set Permaload flag, as required, for an already loaded landblock
+                if (landblock != null)
```

Comment:
This check shouldn't be here. The one you have right after the new Landblock(landblockId) is the correct location.